os/services/rpl-border-router/embedded/Makefile.embedded

```diff
@@ -1,7 +1,11 @@
 PREFIX ?= fd00::1/64
 
 connect-router: $(TUNSLIP6)
+ifeq ($(PORT),)
 	sudo $(TUNSLIP6) $(PREFIX)
+else
+	sudo $(TUNSLIP6) -s $(PORT) $(PREFIX)
```

Comment:
To avoid duplicate code starting with `sudo $(TUNSLIP6)`, defining `TUNSLIP6_FLAGS` would be better like `/arch/cpu/cc26x0-cc13x0/Makefile.cc26x0-cc13x0` does.

Reply:
Thank you for your feedback! I have now addressed it, and defined `TUNSLIP6_FLAGS` as per your suggestion. Does this look okay?